controllers/movement.py

```diff
@@ -189,6 +190,15 @@ def is_at_goal(self) -> bool:
     def manualdrive(self) -> None:
         if self.debug_trajectory:
             self.generate_trajectory()
+
+        # if we arent trying to move and almost at end of a match
+        if (
+            wpilib.DriverStation.getMatchTime() < 5
+            and sum(abs(x) for x in self.driver_inputs) < 0.01
```

Comment:
nit: clearer to use map here
```suggestion
            and sum(map(abs, self.driver_inputs)) < 0.01
```